frameworks/Python/bottle/uwsgi.ini

```diff
@@ -2,7 +2,7 @@
 master
 ; Increase listen queue used for nginx connecting to uWSGI. This matches
 ; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
-listen = 65535
+listen = 128
```

Comment:
From what I can tell, [it appears to be higher on master](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Python/bottle/uwsgi.ini#L5) , which, as far as I know, is what we're using for our Round 14 preview rounds.

Ping @nbrady-techempower @msmith-techempower